src/content/docs/en/guides/troubleshooting.mdx

```diff
@@ -199,6 +199,29 @@ const answer = sum(2, 4);
 <Debug {answer} />
 ```
 
+## Creating minimal reproductions
+
+When you encounter a bug in Astro, it's most helpful to create a minimal reproduction of the issue. This helps us to understand the context, quickly identify the problem, and fix it.
+
+A minimal reproduction should be a simplified version of the code that reliably demonstrates the issue.
+
+### Create a StackBlitz via [astro.new](https://astro.new)
+
+You can use [astro.new](https://astro.new) to create a new Astro project with a single click. For minimal reproductions, we recommend the minimal (empty) example, so you can isolate the issue.
+
+We ask you to use [StackBlitz](https://stackblitz.com) because it allows us to quickly reproduce your issue without setting up a local environment. With a GitHub repository, we would have to clone it, install dependencies, and run it locally, which can introduce more issues.
+You might not be able to reproduce your issue in StackBlitz. For example, your issue might be dependent on a specific environment or package manager, or it may involve HTML Streaming, which isn't supported in StackBlitz. In this case, create a new Astro project using the CLI, reproduce the issue, and upload it to a GitHub repository. Then, link that in the issue.
+### Minimal code
+
+Once your empty project is set up, go through the steps to reproduce the issue. This can include adding packages, changing configuration, and writing code.
+You should only add the minimum amount of code necessary to reproduce the issue. This means that you should remove all code that is not directly related to the issue.
+
+### Create your issue
+
+In most of Astro's repositories you'll find issue templates that will guide you on how to write good issues. It's important that you follow these templates so we don't have to go back and forth with you to get the information we need to solve the issue.
+
+Share the link to your StackBlitz with us in your opened issue. Additionally, include clear, step-by-step instructions on how to replicate the issue and a description of the expected and actual behavior.
```

Comment:
```suggestion
### Create an issue

If your issue can be reproduced, then it is time to create an issue and file a bug report!

Go to the appropriate Astro repository on GitHub and open a new issue. Most repositories have an issue template that will ask questions or require information in order to submit. It's important that you follow these templates because if you don't provide the information we need, then we have to ask you for it... and no one is working on your issue!

Include the link to your minimal reproduction on StackBlitz (or GitHub repository, if necessary). Start with a description of the expected versus actual behavior to provide context for the issue. Then, include clear, step-by-step instructions on how to replicate the issue in an Astro project.
```